src/pages/[platform]/build-a-backend/functions/examples/override_token/index.mdx

```diff
@@ -0,0 +1,109 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Override ID token claims',
+  description:
+    'Use an Auth Pre token generation trigger to override ID token claims',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+
+You can use `defineAuth` and `defineFunction` to create an [Amazon Cognito Pre token generation AWS Lambda trigger](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-pre-token-generation.html) to override the token by adding a new claim or modifying the user's group membership.
+
+<Callout info>
+
+A user is "confirmed" when they verify their account. Typically this happens when the user confirms their email via the verification email. The post confirmation handler will _not_ be triggered for federated sign-ins (i.e. social sign-in).
+
+</Callout>
+
+To get started, create a new directory and a resource file, `amplify/auth/pre-token-generation/resource.ts`. Then, define the function with `defineFunction`:
+
+```ts title="amplify/auth/pre-token-generation/resource.ts"
+import { defineFunction } from '@aws-amplify/backend';
+
+export const preTokenGeneration = defineFunction({
+  name: 'preTokenGeneration',
+});
+```
+
+Then, create the corresponding handler file, `amplify/auth/post-confirmation/pre-token-generation/handler.ts`, file with the following contents:
+
+{/* cSpell:disable */}
+```ts title="amplify/auth/pre-token-generation/handler.ts"
+import type { PreTokenGenerationTriggerHandler } from "aws-lambda";
+
+export const handler: PreTokenGenerationTriggerHandler = async (event) => {
+  event.response = {
+    claimsOverrideDetails: {
+      groupOverrideDetails: {
+        groupsToOverride: ["amplify_group_1"],
```

Comment:
does this add an extra group? can we add a preceding comment highlighting the action?